parsers/s01-parse/crowdsecurity/aws-cloudtrail.md

```diff
@@ -1 +1,18 @@
-Parser for AWS CloudTrail JSON logs
\ No newline at end of file
+# Parser for cloudtrail logs
+
+Parse with the cloudtrail parser logs with type `aws-cloudtrail`. You may want to use the transform feature of crowdsec with the following configuration: `map(JsonExtractSlice(evt.Line.Raw, "Records"), ToJsonString(#))`.
```

Comment:
this is true only if you use s3 ? maybe we should specify both cases (s3 and kinesis) + link to the acquis doc